src/components/Molecules/SearchInput/SearchInput.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Input from '../../Atoms/Input/Input';
+import spacing from '../../../theme/shared/spacing';
+import searchIcon from './assets/search.svg';
+
+const Wrapper = styled.div`
+  width: 100%;
+  max-width: 1200px;
+  margin: auto;
+`;
+
+const InnerWrapper = styled.div`
+  margin: ${spacing('md')} auto;
+  padding: 0 calc((4 * ${spacing('xsm')}));
+`;
+
+const Form = styled.form`
+  display: flex;
+  margin: ${spacing('xl')} auto;
+  width: 100%;
+  border-bottom: 4px solid ${({ theme }) => theme.color('red')};
+  overflow: hidden;
+`;
+
+const SearchWrapper = styled.div`
+  width: 80%;
+  margin: ${spacing('md')} ${spacing('md')} 0;
+  @media ${({ theme }) => theme.breakpoint('medium')} {
+    margin-bottom: calc(${spacing('md')} - ${spacing('xsm')});
```

Comment:
let's try to use a single var.. maybe in this case `l` is enough.

Reply:
Or maybe just `md`? I wouldn't worry about it matching the current design exactly; with the new rem spacing, it's never gonna be _that_ close, so maybe best just use whichever of our range is close enough?

Reply:
that's true

Reply:
Done